commands/keys.go

```diff
@@ -0,0 +1,192 @@
+/*
+Copyright 2018 The Doctl Authors All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package commands
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/digitalocean/doctl"
+	"github.com/digitalocean/doctl/commands/displayers"
+	"github.com/digitalocean/doctl/do"
+	"github.com/spf13/cobra"
+)
+
+// Keys generates the serverless 'keys' subtree for addition to the doctl command
+func Keys() *Command {
+	cmd := &Command{
+		Command: &cobra.Command{
+			Use:   "key",
+			Short: "Manage access keys for functions namespaces",
+			Long: `Access keys provide secure authentication for serverless operations without using your main DigitalOcean token.
+
+These commands allow you to create, list, and delete namespace-specific access keys.
+Keys operate on the currently connected namespace by default, but can target any namespace using the --namespace flag.`,
+			Aliases: []string{"keys"},
+		},
+	}
+
+	create := CmdBuilder(cmd, RunAccessKeyCreate, "create", "Creates a new access key",
+		`Creates a new access key for the specified namespace. The secret is displayed only once upon creation.
+
+Examples:
+  doctl serverless key create --name "my-laptop-key"
+  doctl serverless key create --name "ci-cd-key" --namespace fn-abc123`,
+		Writer)
+	AddStringFlag(create, "name", "n", "", "name for the access key", requiredOpt())
+	AddStringFlag(create, "namespace", "", "", "target namespace (uses connected namespace if not specified)")
+
+	list := CmdBuilder(cmd, RunAccessKeyList, "list", "Lists access keys",
+		`Lists all access keys for the specified namespace with their metadata.
+
+Examples:
+  doctl serverless key list
+  doctl serverless key list --namespace fn-abc123`,
+		Writer, aliasOpt("ls"), displayerType(&displayers.AccessKeys{}))
+	AddStringFlag(list, "namespace", "", "", "target namespace (uses connected namespace if not specified)")
+
+	delete := CmdBuilder(cmd, RunAccessKeyDelete, "delete <key-id>", "Deletes an access key",
+		`Permanently deletes an existing access key. This action cannot be undone.
+
+Examples:
+  doctl serverless key delete dof_v1_a1b2c3d4e5f67890
```

Comment:
Is it possible to specify <access_key_id> or equivalent in description or one of the examples?